exporter/export.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import "sync"
+
+var (
+	exportersMu   sync.RWMutex // guards exporters
+	viewExporters = make(map[View]struct{})
+)
+
+// View exports the collected view data.
+//
+// The ExportView method should return quickly; if an
+// Exporter takes a significant amount of time to
+// process a ViewData, that work should be done on another goroutine.
+//
+// The ViewData should not be modified.
+type View interface {
+	ExportView(viewData *ViewData)
+}
+
+// Register registers an exporter. The exporter should implement one of the
```

Comment:
How about:
`The exporter should implement the View interface`
as there is only one export interface.